src/app/views/media/clusters.jsx

```diff
@@ -31,11 +46,92 @@ const UserList3 = () => {
         setPage(0);
     };
 
-    useEffect(async () => {
-        const resp = await bc.registry().getAllClusters({ limit: rowsPerPage, offset: page * rowsPerPage });
-        if (resp.status == 200) setClusters(resp.data)
+    const handleSearch = (value) => {
```

Comment:
We have a hook to get the query from the url, here is the code: https://github.com/avanegasp/breatheco-admin/blob/c0b34b179c8938919fa62c9d58338ef76f749cc8/src/app/views/media/assets.jsx#L62